frontend/app/src/components/form/fields/common.tsx

```diff
@@ -12,6 +12,7 @@ type LabelFormFieldProps = {
   required?: boolean;
   unique?: boolean;
   description?: string;
+  variant?: string;
```

Comment:
We lost typing here, in case like this prefer:
```ts
interface LabelFormFieldProps extends LabelProps {
  className?: string;
  label?: string;
  required?: boolean;
  unique?: boolean;
  description?: string;
}
```
So you have access to every props of primitive component